### PR TITLE
PR for results page! 

### PR DESCRIPTION
**Pull request by adidoshi**
### Hey I created result page for quiz app!
* [Live preview](https://deploy-preview-4--boring-elion-77cf1f.netlify.app/screens/result/result)

### CODE REVIEW

**Review by siddhigate — APPROVED** [review]
UI looks good! Code is well structured. Good to merge.

File: screens/result/Result.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+.result-title p span {
+  color: var(--primary-color);
+  font-weight: bold;
+}
+
+.result-hr {
+  border-color: var(--dark-gray);
+  margin: 1.4rem 0 0.4rem 0;
```
Comment (msdeshmukh009) [inline screens/result/Result.css:21]: Suggestion: Try to keep the margin and padding in multiples of 8px/0.5rem or whatever you prefer just try to keep it consistent.

**Review by msdeshmukh009 — APPROVED** [review]
Rest is looking good.

File: screens/result/Result.css (hunk 1 above)
Comment (adidoshi) [reply screens/result/Result.css:21]: Sure! I'll do that.

**Pull request merged**